.eslintrc.js

```diff
@@ -312,5 +317,11 @@ module.exports = {
         '@typescript-eslint/explicit-module-boundary-types': OFF,
       },
     },
+    {
+      files: ['*.test.ts', '*.test.tsx'],
+      rules: {
+        'import/no-extraneous-dependencies': OFF,
```

Comment:
maybe this should be disabled with more granularity (if possible)? 🤷‍♂️

Reply:
Test files are internal anyways. If they can run, there's nothing to worry about.